
# Frame Parameters
odom_frame: 'map'
base_frame: 'base_footprint'

#Bool

use_servo_cmd_to_calc_angular_velocity: true
publish_tf: false 

#Linear Speed Calculation Parameters

speed_offset: 16.6
speed_gain: 3485.3528

#Angular Speed Calculation Parameters

steering_angle_to_servo_gain: 0.015
steering_angle_to_servo_offset: 0.45

wheelbase: 0.28
